src/installation/index.md

```diff
@@ -1,117 +1,129 @@
 # Setting Up a Development Environment
 
-With an understanding of the ecosystem surrounding Rust on Espressif chips, we can move on to actual development. If you are not aware of the two possible development approaches or do not understand the differences between writing `std` and `no_std` applications, please first read the [Ecosystem Overview] chapter.
+At the moment, Espressif SoCs are based on two different architectures -- `RISC-V` and `Xtensa`. Each architecture supports the `std` and `no_std` application development.
 
-Let's take a moment to discuss the Rust support for the different architectures of the Espressif chips and the different approaches in more detail. At this moment, Espressif SoCs are based on two different architectures: `RISC-V` and `Xtensa`. The support for those two architectures in the Rust programming language is very different.
+To set up the development environment, do the following:
 
-[Ecosystem Overview]: ../overview/index.md
+1. [Install Rust][install-rust]
+2. Install requirements for the targets
+    - [RISC-V targets][risc-v-targets]
+    - [Xtensa targets][xtensa-targets]
 
-## RISC-V targets
+As mentioned in the installation procedures below, for `std` development also don't forget to install [`std` Development Requirements][rust-esp-book-std-requirements].
 
-The `RISC-V` architecture has support in the mainline Rust compiler so, the setup is relatively simple. There are two ways of proceeding with the installation:
-- Using the official Rust tools
-- Using [`espup`, a tool that will be covered later]
+Please note that you can host the development environment with support for Xtensa in a [container][use-containers].
```

Comment:
We also provide containers for RISC-V targets

Reply:
Got it. Suggest updating the container description on the [idf-rust](https://hub.docker.com/r/espressif/idf-rust/tags) page.